### PR TITLE
Pull latest raft updates

### PR DESCRIPTION
**Pull request by michelvocks**

### CODE REVIEW

**Comment by mladlow** [comment]
We talked about this in sprint planning and thought the raft library might be the best place for this metric to live - would you mind adding it there? Else we will add it next sprint for inclusion in Vault 1.5.5.

**Comment by michelvocks** [comment]
@mladlow Thanks! I've created https://github.com/hashicorp/raft/pull/431. I will leave this open to remind me that we should update the docs in case https://github.com/hashicorp/raft/pull/431 gets merged.

**Comment by mladlow** [comment]
Sounds good - we'll also need to update and backport the Vault dependency on the Raft library when the raft PR is merged.

**Comment by michelvocks** [comment]
@mladlow https://github.com/hashicorp/raft/pull/431 has been merged and I've updated this PR with the dependency update.

File: website/pages/docs/internals/telemetry.mdx [hunk 1]
```diff
@@ -416,7 +417,7 @@ indicate flapping leadership.
 
 ## Metric Labels
 
-| Metric                                          | Description                                                                | Example                            | 
+| Metric                                          | Description                                                                | Example                            |
```
Comment (ncabatoff) [inline website/pages/docs/internals/telemetry.mdx:420]: Can we add peer_id to this table?

Comment (michelvocks) [reply website/pages/docs/internals/telemetry.mdx:420]: @ncabatoff Thanks! Done

**Review by ncabatoff — APPROVED** [review]

**Comment by michelvocks** [comment]
@mladlow Let me know if there is some backport work to do.

**Comment by mladlow** [comment]
@michelvocks Could you please backport to release/1.5.x? We'll need it there to include it in 1.5.5. Thanks!

**Pull request merged**